analyzers/src/SonarAnalyzer.CSharp/Rules/SymbolicExecution/SymbolicExecutionRunner.cs

```diff
@@ -56,11 +57,19 @@ public class SymbolicExecutionRunner : SymbolicExecutionRunnerBase
             new InitializationVectorShouldBeRandom(),
             new HashesShouldHaveUnpredictableSalt());
 
-        protected override ImmutableDictionary<DiagnosticDescriptor, RuleFactory> AllRules { get; } = ImmutableDictionary<DiagnosticDescriptor, RuleFactory>.Empty
+        public SymbolicExecutionRunner() : this(AnalyzerConfiguration.AlwaysEnabled)
```

Comment:
This one should call base
```suggestion
        public SymbolicExecutionRunner() : base(AnalyzerConfiguration.AlwaysEnabled)
```